src/ui/CustomStyleEditor.js

```diff
@@ -1428,6 +1435,7 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
     const hiddenDiv = document.getElementById('nextStyle');
     if (hiddenDiv && hiddenDiv.style) {
       hiddenDiv.style.display = display;
+      hiddenDiv.style.marginBottom = '-7px';
```

Comment:
Is 7 a fraction of a font size? This looks like it should be in em instead of px.